prinz_pmml/src/main/scala/pl/touk/nussknacker/prinz/pmml/PMMLTest.scala

```diff
@@ -0,0 +1,31 @@
+package pl.touk.nussknacker.prinz.pmml
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.scalalogging.LazyLogging
+import pl.touk.nussknacker.prinz.pmml.repository.PMMLModelRepository
+
+import java.io.{BufferedReader, InputStreamReader}
+
+// TODO to be removed after full repository implementation
+object PMMLTest extends App with LazyLogging {
+  val d1  = "file:///home/avan1235/Desktop/pmml/fraud-detection.pmml"
+  val d2  = "file:///home/avan1235/Desktop/pmml/"
```

Comment:
`/home` dir. Is this PR wip?

Reply:
I would name it partial PR to make it posiible for other guys to work with it. Here you can see the local directory usage of repository for pmml that is working

Reply:
How about pulling into a wip branch and merging to `master` once it's ready?